browser/extensions/brave_component_loader.cc

```diff
@@ -108,27 +98,6 @@ void BraveComponentLoader::AddDefaultComponentExtensions(
   UpdateBraveExtension();
 }
 
-#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED)
-void BraveComponentLoader::AddEthereumRemoteClientExtension() {
-  AddExtension(kEthereumRemoteClientExtensionId,
-               kEthereumRemoteClientExtensionName,
-               kEthereumRemoteClientExtensionPublicKey);
```

Comment:
Should we delete this extension from disk?
Something like https://github.com/brave/brave-core/blob/master/components/ipfs/ipfs_component_cleaner.cc#L15-L21
If yes, needs a follow up issue